### PR TITLE
add method reply_dice and change send_dice docs.

### PR DESCRIPTION
**Pull request by jonatan1609**

### CODE REVIEW

File: pyrogram/methods/messages/send_dice.py [hunk 1]
```diff
@@ -47,8 +47,9 @@ async def send_dice(
                 For a contact that exists in your Telegram address book you can use his phone number (str).
 
             emoji (``str``, *optional*):
-                Emoji on which the dice throw animation is based. Currently, must be one of "🎲",  "🎯", "🏀" or "⚽️".
-                Defaults to "🎲".
+                Emoji on which the dice throw animation is based.
+                Currently, must be one of :obj:`~pyrogram.emoji.SOCCER_BALL`,  :obj:`~pyrogram.emoji.BASKETBALL`, :obj:`~pyrogram.emoji.DIRECT_HIT` or :obj:`~pyrogram.emoji.GAME_DIE`.
```
Comment (delivrance) [inline pyrogram/methods/messages/send_dice.py:51]: Emoji references don't seem to play nice with docs. We'd better revert to what it was for now.

**Review by delivrance — CHANGES_REQUESTED** [review]

**Comment by jonatan1609** [comment]
I've changed them back

File: pyrogram/types/messages_and_media/message.py [hunk 2]
```diff
@@ -2732,32 +2732,16 @@ async def forward(
...
-                    file_id = self.voice.file_id
-                    file_ref = self.voice.file_ref
-                elif self.sticker:
-                    file_id = self.sticker.file_id
-                    file_ref = self.sticker.file_ref
-                elif self.video_note:
-                    file_id = self.video_note.file_id
-                    file_ref = self.video_note.file_ref
-                elif self.contact:
+                file_id, file_ref = None, None
```
Comment (delivrance) [inline pyrogram/types/messages_and_media/message.py:2735]: Please revert this, it has nothing to do with dice. You can open a new PR with your proposed changes (the relevant code can now be found in `async def copy`).

File: pyrogram/methods/messages/send_dice.py [hunk 3]
```diff
@@ -47,9 +47,9 @@ async def send_dice(
                 For a contact that exists in your Telegram address book you can use his phone number (str).
 
             emoji (``str``, *optional*):
-                Emoji on which the dice throw animation is based. Currently, must be one of "🎲",  "🎯", "🏀" or "⚽️".
+                Emoji on which the dice throw animation is based.
+                Currently, must be one of "🎲",  "🎯", "🏀" or "⚽️".	                Emoji on which the dice throw animation is based.
```
Comment (delivrance) [inline pyrogram/methods/messages/send_dice.py:51]: There's also 🎰  now.

**Pull request closed without merging**